### DISCUSSION TITLE
lora library that show transmitter on air time

### DISCUSSION THREAD

**Discussion by kjm1102** [micropython/micropython#11607] [Libraries & Drivers]
When I was using LoPy4s there was a nice feature baked into pycom's version of upython called lora.stats that could give some useful parameters, I was particularly fond of the 'on-air' statistic that gave the transmitter active time. I can see the source code behind the stats in https://github.com/pycom/pycom-micropython-sigfox/blob/4481a1c283be97640592ea2a0440830e336064de/esp32/mods/modlora.c#L821 but I don't have the smarts to roll C code into my own custom upython.

These days, with the demise of pycom, I'm using https://github.com/lemariva/uPyLoRaWAN for raw lora on a TTGO board but the transmit interval is not available. Just wondering if anybody has spied a upython lora lib in their travels that offers transmitter on air time as a statistic?

**Comment by davefes** [comment]
https://github.com/ehong-tl/micropySX126X
has a get time on air method
